backend/stylists/app/Http/Controllers/ProductController.php

```diff
@@ -129,9 +129,9 @@ public function getCreate(Request $request)
             'desc' => 'required|min:5',
             'merchant' => 'required|min:4',
             'brand' => 'required|min:2',
-            'category' => 'required|min:2',
```

Comment:
@avinashpatel57 category, gender, color1 should be required. This code is called by chrome extension. The stylist must enter all 3 values for adding a product.